### PR TITLE
Update SearchOptionServlet.java

### PR DESCRIPTION
**Pull request by tmatsuo**
Corrected the position of an end region tag.

### CODE REVIEW

**Comment by tmatsuo** [comment]
@lesv Merging. Please put comments if you have any concerns

**Comment by tswast** [comment]
LGTM.

File: appengine/search/src/main/java/com/example/appengine/search/SearchOptionServlet.java [hunk 1]
```diff
@@ -95,8 +95,8 @@ private Index getIndex() {
     } catch (SearchException e) {
       // handle exception...
```
Comment (lesv) [inline appengine/search/src/main/java/com/example/appengine/search/SearchOptionServlet.java]: For the record, we really shouldn't be doing this.  We should actually handle the exception or propagate it.

Comment (tmatsuo) [inline appengine/search/src/main/java/com/example/appengine/search/SearchOptionServlet.java]: The comment says "handle exception", and it is deferred to users how to handle it :)

Comment (lesv) [inline appengine/search/src/main/java/com/example/appengine/search/SearchOptionServlet.java]: Yes - that is the point, we need to show them how to handle it properly.

**Pull request merged**